### DISCUSSION TITLE
Problems :(

### DISCUSSION THREAD

**Discussion by Turtlecodez** [nroutasuo/level13#274] [Q&A]
(I've played this game before, about a year ago) I have a big hairpin problem. I've just started, and I'm still about halfway through Level 13. I have a problem with hairpins. I haven't unlocked crafting yet, and I'm not equipped for any fight. What should I do?
(I also have a source of leather right next to my camp,which most people would die for late game)

**Comment by nroutasuo** [comment]
Can you share your world seed? I can check that it's not entirely broken regarding hairpins. 

The next version will make it a little less random to find resources like this (it will at least tell you if there are crafting resources to be found when scouting already, instead of randomly when scavenging).

**Reply by Turtlecodez** [reply]
Sorry, I forgot to follow up. I randomly found some ragged pants while scavenging, which somehow made the deadly pests fairly harmless. I’m now on Level 12 and still progressing. It’s all ok now!

**Reply by Turtlecodez** [reply]
I probably just forgot how hard early game was lol
